### PR TITLE
test(react-wildcat-prefetch): add missing tests

### PR DESCRIPTION
**Pull request by doctyper**
add tests for prefetching multiple components

### CODE REVIEW

File: packages/react-wildcat-prefetch/src/test/browser/prefetchSpec.js [hunk 1]
```diff
@@ -324,6 +324,64 @@ describe("react-wildcat-prefetch", () => {
...
+                        firstData: {}
+                    }
+
+                    render() {
+                        expect(this.props.firstData)
+                            .to.eql(stubs.prefetchedData.firstData);
+
+                        if (renderCount > 1) {
+                            done();
+                        }
```
Comment (dhoward) [inline packages/react-wildcat-prefetch/src/test/browser/prefetchSpec.js]: Why does the component need to render twice before the props are passed?

File: packages/react-wildcat-prefetch/src/test/browser/prefetchSpec.js [hunk 2]
```diff
@@ -324,6 +324,64 @@ describe("react-wildcat-prefetch", () => {
...
+                            .to.eql(stubs.prefetchedData.secondData);
+
+                        if (renderCount > 1) {
+                            done();
+                        }
+
+                        renderCount++;
+                        return <div>{Object.keys(this.props)}</div>;
+                    }
+                }
```
Comment (dhoward) [inline packages/react-wildcat-prefetch/src/test/browser/prefetchSpec.js]: These 2 components are the same except for `props.firstData` and `props.secondData`. Maybe could just use one class and pass that as a prop? `<WrappedPrefetch title={"Test Title"} prefetchedData={"firstData"} />`. Would use the static prop validation but I'm not sure you need it here.

File: packages/react-wildcat-prefetch/src/test/browser/prefetchSpec.js [hunk 3]
```diff
@@ -469,6 +527,82 @@ describe("react-wildcat-prefetch", () => {
...
+                            if (totalRenderCount > 1) {
+                                done();
+                            }
+
+                            totalRenderCount++;
+                        }
+
+                        return <div>{Object.keys(this.props)}</div>;
+                    }
+                }
```
Comment (dhoward) [inline packages/react-wildcat-prefetch/src/test/browser/prefetchSpec.js]: Same as above

File: packages/react-wildcat-prefetch/src/test/browser/prefetchSpec.js (hunk 1 above)
Comment (doctyper) [inline packages/react-wildcat-prefetch/src/test/browser/prefetchSpec.js]: That's not what's happening here. Each component is rendering once, but I'm only calling `done()` when both have rendered.

Comment (dhoward) [inline packages/react-wildcat-prefetch/src/test/browser/prefetchSpec.js]: Got it, misunderstood what was going on

File: packages/react-wildcat-prefetch/src/test/browser/prefetchSpec.js (hunk 2 above)
Comment (doctyper) [inline packages/react-wildcat-prefetch/src/test/browser/prefetchSpec.js]: I could, but I would rather make sure each component is independent of each other to make this test pristine.

Comment (dhoward) [inline packages/react-wildcat-prefetch/src/test/browser/prefetchSpec.js]: Fair enough

**Pull request merged**